### ISSUE TITLE
MacOS Catalina - Can't tap iina/homebrew-mpv-iina

### ISSUE THREAD

**Issue by Napoleon-BlownApart** [iina/homebrew-mpv-iina#2]
[MacOS Catalina 10.15.3]

Can't tap this repo, see error below:

    brew tap iina/homebrew-mpv-iina
    ==> Tapping iina/mpv-iina
    Cloning into '/usr/local/Homebrew/Library/Taps/iina/homebrew-mpv-iina'...
    remote: Enumerating objects: 38, done.
    remote: Counting objects: 100% (38/38), done.
    remote: Compressing objects: 100% (26/26), done.
    remote: Total 125 (delta 18), reused 31 (delta 12), pack-reused 87
    Receiving objects: 100% (125/125), 44.74 KiB | 192.00 KiB/s, done.
    Resolving deltas: 100% (58/58), done.
    Error: Invalid formula: /usr/local/Homebrew/Library/Taps/iina/homebrew-mpv-iina/ffmpeg-iina.rb
    ffmpeg-iina: undefined method `use_from_macos' for #<Class:0x00007fa9281c4e58>
    Error: Cannot tap iina/mpv-iina: invalid syntax in tap!

**Comment by inflation** [comment]
Not happening in my tests. May be network issue?

**Comment by Napoleon-BlownApart** [comment]
I'm sorry for not updating this issue earlier. I raised it on the `inna` repo and they fixed it.  From the response, I got the impression that you were aware.  So the `use_from_macos` error does not occur anymore.   https://github.com/iina/iina/issues/2793#issue-573556122

There are some further build issues, but I haven't had a chance to investigate them further.  Something to do with a library that I already have installed.

I will close this.  Thanks.
